modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/context/RequestContextMapperImpl.java

```diff
@@ -151,6 +154,21 @@ public Context map(HttpServletRequest httpServletRequest) {
 			Context.REQUEST_PARAMETERS,
 			_getRequestParameters(httpServletRequest));
 
+		try {
```

Comment:
Finally I'm using the mapper, I think It's the right place. But, what do you think about not being declaring a key for "segmentsEntryIds"? A think we can address it later, but maybe you have a different thought

Reply:
A new key is not a breaking change, right?

Reply:
It's a minor change

Reply:
I'm going to add it ;)